pretyping/globEnv.ml

```diff
@@ -33,6 +33,8 @@ type t = {
   (** For locating indices *)
   renamed_env : env;
   (** For name management *)
+  renamed_vars : EConstr.t list Lazy.t;
+  (** Identity instance of named part of the renamed environment *)
```

Comment:
Could you please write in the comment that this is a cache to maximize sharing?

Reply:
While it is pretty clear in the context of this PR what this is for, the comment is too vague IMO. Not the the one above is clear ;-) but at least the new one can be improved

Reply:
Done.